app/models/watched_word.rb

```diff
@@ -8,9 +8,10 @@ def self.actions
       censor: 2,
       require_approval: 3,
       flag: 4,
+      link: 8,
```

Comment:
I think we should order the hash here by the number representing the action code. Another developer might come along and incorrectly use `8` if they do not notice that it has already exists.

Reply:
The wrong order here is on purpose, because we want "Link" above "Replace". I can add a comment saying that 9 is the ID of the next watched word type. Will that work?

Reply:
Hmm is somewhere in the code depending on the specific order of the keys in the hash?

Reply:
The server serializes the actions and then the client renders them in exact same order.

Reply:
ooo icic. Ideally, the UI wouldn't depend on the ordering of code in the backend but I guess the ship has already sailed on this one. I guess we can punt on this for now :) Thank you for the explaination.